src/rgw/rgw_user.h

```diff
@@ -57,6 +57,23 @@ extern int rgw_user_get_all_buckets_stats(RGWRados *store, const rgw_user& user_
  */
 extern void rgw_get_anon_user(RGWUserInfo& info);
 
+/**
+ * Add group name to user info
+ */
+extern bool rgw_add_group_to_user(RGWUserInfo& info, const string& group_name);
```

Comment:
We should probably stop sticking `extern` on function prototypes since it's the default.